src/Rpc.proto

```diff
@@ -197,11 +197,16 @@ message SetPeerBlackListResponse{
 }
 
 message GetPeerInfoRequest {
-    bool query = 1;
+    bytes publicKey = 1;
+    bytes ip = 2;
 }
 
 message GetPeerInfoResponse {
-    string query = 1;
+    bytes clientId = 1;
```

Comment:
if you are meaning to return a PeerId, shouldn't you simply use a PeerId field to store the info?

Reply:
Yes, you are correct. I thought I could only use simple types but I can see complex types being used elsewhere so I'll do that. 

In the case of the repeated PeerId. That means there is no unique identifier for a node? so nodes with the same PeerId are all affected by get/update/delete? Ok, i'll add that too. Cheers.

Reply:
I don't know the context of this change to be honest, just that to identify uniquely a peer, it is best to use the PeerId, rather than a subset of its fields :)
maybe check with @nshCore what he thinks